compose/backend.yml

```diff
@@ -131,19 +131,21 @@ services:
     networks:
       - ccd-network
 
-#  am-database:
-#    image: hmcts.azurecr.io/am-db
-#    container_name: access_management_db
-#    environment:
-#      AM_DB:
-#      AM_DB_USERNAME:
-#      AM_DB_PASSWORD:
-#    healthcheck:
-#      interval: 10s
-#      timeout: 10s
-#      retries: 10
-#    ports:
-#      - 5600:5432
+  am-database:
+    image: hmctspublic.azurecr.io/am/db:latest
+    container_name: access_management_db
+    environment:
+      AM_DB:
+      AM_DB_USERNAME:
+      AM_DB_PASSWORD:
+    healthcheck:
+      interval: 10s
+      timeout: 10s
+      retries: 10
+    ports:
+      - 5600:5432
+    volumes:
+      - am-database-data:/var/lib/postgresql/data
```

Comment:
I think this part should go under others.yml as per the latest ccd-docker design, at least until am integration actually becomes and integral part of ccd.

Reply:
Moreover, there should be a networks attribute as is the case with similar other definitions.